packages/kodeverk/src/aksjonspunktCodes.ts

```diff
@@ -40,12 +40,12 @@ enum AksjonspunktCode {
   VURDERE_ANNEN_YTELSE = '5033',
   VURDERE_DOKUMENT = '5034',
   BEHANDLE_KLAGE_NFP = '5035',
-  BEHANDLE_KLAGE_NK = '5036',
+  BEHANDLE_KLAGE_NK = '5036', // Utgått, fjerne bruk i panel?
   VURDER_INNSYN = '5037',
   REGISTRER_PAPIRSOKNAD_FORELDREPENGER = '5040',
   VURDER_ARBEIDSFORHOLD_PERMISJON = '5041',
   VURDER_SOKNADSFRIST_FORELDREPENGER = '5043',
-  KONTROLLER_AUTOMATISK_BESTEBEREGNING = '5048',
+  KONTROLLER_AUTOMATISK_BESTEBEREGNING = '5048', // Utgått, brukes i visning?
```

Comment:
Stemmer, viser gamle saker som fikk dette istedenfor det nye (5062)

Reply:
@pekern kanskje du kunne tatt en titt på hvilke av de som er merket // Utgått som kan fjernes
Jeg ser de som brukes i historikk(Mal3) og noe rundt totrinn (som kanskje kan fjernes) - men ikke sikker på resten.
Det skal ikke være noen åpne behandlinger med disse - bare avsluttede - men usikker på hva som trengs for å vise gamle behandlinger og enable alle relevante panel for disse